tests/core/search_indices_bis.py

```diff
@@ -24,6 +24,15 @@ class UserIndex(ModelIndex):
     effective_date = DateField(eval_as='obj.created if obj.created and obj.published > obj.created else obj.published')
     meta_data = StringField(eval_as='" ".join([fld for fld in [obj.link, str(obj.tweet_count), obj.raw] if fld])')
     more_fields = StringField(eval_as='"some value"')
+    int_description = NumberField(coretype='integer')
+
+    def prepare_int_description(self, obj):
```

Comment:
This function is actually called in the tests: https://coveralls.io/builds/3234838/source?filename=tests%2Fcore%2Fsearch_indices_bis.py#L29 .